lms/envs/production.py

```diff
@@ -619,7 +619,7 @@ def get_env_setting(setting):
         'social_core.backends.linkedin.LinkedinOAuth2',
         'social_core.backends.facebook.FacebookOAuth2',
         'social_core.backends.azuread.AzureADOAuth2',
-        'social_core.backends.apple.AppleIdAuth',
+        'third_party_auth.appleid.AppleIdAuth',  # vendored 'social_core.backends.apple.AppleIdAuth'
```

Comment:
Retrospective note: This was later changed to include the `common.djangoapps.` prefix as part of the effort to remove the sys path hacks.